Source/Core/DolphinWX/GameListCtrl.h

```diff
@@ -14,6 +14,9 @@
 
 #include "DolphinWX/ISOFile.h"
 
+// Luminance threshold to determine whether to use black text on light background
+#define LUM_THRESHOLD 186
```

Comment:
Like I had explained in my initial comment about the constant. This can be a local `constexpr` variable within the function itself. Please [do not use defines for constants or 'functions'](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es31-dont-use-macros-for-constants-or-functions).